big_tests/run_common_test.erl

```diff
@@ -1,18 +1,18 @@
 %% During dev you would use something similar to:
-%% TEST_HOSTS="mim" ./tools/travis-test.sh -e false -c false -s false -p odbc_mssql_mnesia
+%% TEST_HOSTS="mim1" ./tools/travis-test.sh -c false -s false -p odbc_mssql_mnesia
 %%
 %% If you also want to start just mim1 node use:
-%% DEV_NODES="mim1" TEST_HOSTS="mim" ./tools/travis-test.sh -e false -c false -s false -p odbc_mssql_mnesia
+%% DEV_NODES="mim1" TEST_HOSTS="mim" ./tools/travis-test.sh -c false -s false -p odbc_mssql_mnesia
 %%
 %% TEST_HOSTS variable contains host names from hosts in big_tests/test.config.
-%% DEV_NAMES variable contains release names from profiles in rebar.config.
+%% DEV_NODES variable contains release names from profiles in rebar.config.
 %% Release names are also used to name directories in the _build directory.
 %%
 %% Valid TEST_HOSTS are mim, mim2, mim3, fed, reg.
-%% Valid DEV_NAMES are mim1, mim2, mim3, fed1, reg1.
+%% Valid DEV_NODES are mim1, mim2, mim3, fed1, reg1.
 %%
 %% Example with two nodes:
-%% DEV_NODES="mim1 mim2" TEST_HOSTS="mim mim2" ./tools/travis-test.sh -e false -c false -s false -p odbc_mssql_mnesia
+%% DEV_NODES="mim1 mim2" TEST_HOSTS="mim mim2" ./tools/travis-test.sh -c false -s false -p odbc_mssql_mnesia
```

Comment:
This does not seem to be consistent with modified line 2 above. Here we have `TEST_HOSTS="mim mim2"`, but above there's a change from `TEST_HOSTS="mim"` to `TEST_HOSTS="mim1"` - shouldn't we follow the example here, too?

Reply:
fixed